packages/framework/src/Foundation/Kernel/Hyperlinks.php

```diff
@@ -100,28 +102,30 @@ public function mediaLink(string $destination, bool $validate = false): string
             throw new FileNotFoundException($sourcePath);
         }
 
-        return $this->withCacheBusting($this->relativeLink("{$this->kernel->getMediaOutputDirectory()}/$destination"), $destination);
+        return static::withCacheBusting($this->relativeLink("{$this->kernel->getMediaOutputDirectory()}/$destination"), $destination);
     }
 
     /**
      * Gets a relative web link to the given file stored in the `_site/media` folder.
      * If the image is already qualified (starts with `http`) it will be returned as is.
```

Comment:
This feature doesn't really make sense, why would we pass a value just to have it be returned the same? Are there (valid) usages where it is not known if the input is for a local or remote file?

Reply:
Removing this may cause this:

```
Hyde\Framework\Exceptions\FileNotFoundException: File [_media/https://example.com/foo] not found when trying to resolve a media asset.
```

If we keep this we could add an upgrade helper to have a more helpful message if this is detected

Reply:
Removed in be2e212318cc52a12f83ad9e2ee5c63da85a1e4d